block.go

```diff
@@ -71,3 +74,20 @@ func DeserializeBlock(d []byte) *Block {
 
 	return &block
 }
+
+func (b *Block) PrintHTML(detail bool) string {
+	var lines []string
+	lines = append(lines, fmt.Sprintf("<h2>Block <a href=\"/block/%x\">%x</a> </h2>", b.Hash, b.Hash))
```

Comment:
It's better to do this via 'html/template', and this should be a separate package, because it's not directly related to the logic of the blockchain.

Reply:
Agree, I just learn go, so...